_posts/2018-07-31-why-emacs-is-a-great-editor.md

```diff
@@ -0,0 +1,49 @@
+---
+layout:     post
+title:      Why Emacs is a great text editor
+date:       2018-07-31
+summary:    My thoughts on Emacs
+categories: emacs
+---
+
+![swiss-knife](https://i.imgur.com/J5Od0Xm.png)
+
+Often arguments arise from the fact that I use emacs as my main text editor. In this post, I'll try to explain why I think emacs is a great text editor and how it helps me be more productive every day.
+
+### History
+
+Before using emacs I've tried different text editors (Atom, Brackets etc) but I've primarily used Sublime for a couple of years before switching to emacs. There are two fundamental reasons that motivated me to learn it.
```

Comment:
I think you could switch `I've` with `I` twice in this sentence and it would read better, 

> Before using emacs I tried different text editors

> I primarily used Sublime